packages/zoe/test/unitTests/contracts/test-publicSwap.js

```diff
@@ -7,12 +7,11 @@ import { setup } from '../setupBasicMints';
 
```

Comment:
Seems like a mistake for `bundleSource` to be importable. It looks like it has a huge amount of ambient authority. Can it read my whole disk? Is it visible only to test code?